Source/items.cpp

```diff
@@ -4247,9 +4243,9 @@ static void SpawnOnePremium(int i, int plvl, int myplr)
 	int strength = std::max(myPlayer.GetMaximumAttributeValue(CharacterAttribute::Strength), myPlayer._pStrength);
 	int dexterity = std::max(myPlayer.GetMaximumAttributeValue(CharacterAttribute::Dexterity), myPlayer._pDexterity);
 	int magic = std::max(myPlayer.GetMaximumAttributeValue(CharacterAttribute::Magic), myPlayer._pMagic);
-	strength *= 1.2;
-	dexterity *= 1.2;
-	magic *= 1.2;
+	strength += strength / 5;

```

Comment:
Was there a particular reason for changing this from float multiplication to int division? I don't think int division causes any loss of precision, mainly commenting as I'm curious.

Reply:
The result is the same, but it avoid floats which isn't ideal on some systems. It was done after a comment by @thebigMuh https://github.com/diasurgical/devilutionX/pull/2055#discussion_r639114212